Library/Homebrew/download_strategy.rb

```diff
@@ -389,6 +389,23 @@ def _fetch
   end
 end
 
+# Will convert "https://downloads.sourceforge.net/project/..." URLs to mirrorservice.org
+# In case environment variable HOMEBREW_NO_INSECURE_REDIRECT is set, so the Sourceforge is not ultimately broken
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    unless ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
```

Comment:
Have you fixed the problem in https://github.com/Homebrew/homebrew/pull/42955#issuecomment-132358467 ?

Yes it would be great for you to rebase the commits.